docker-compose.yaml

```diff
@@ -116,6 +116,7 @@ services:
       - SYNC_JOB_MAX_TIMEOUT_DAYS=${SYNC_JOB_MAX_TIMEOUT_DAYS}
       - TEMPORAL_HOST=${TEMPORAL_HOST}
       - TRACKING_STRATEGY=${TRACKING_STRATEGY}
+      - WEBAPP_URL=${WEBAPP_URL}
```

Comment:
where is this coming from?

Reply:
This is needed to initialize the jobNotifier call by the temporal activity

Reply:
This will also need to be updated for Kustomize/Helm in the future.

This is probably beyond the scope for this PR but for internal uses like this we should probably aim at using `INTERNAL_API_HOST` + `API_URL` and not `WEBAPP_URL`.

Reply:
what about the kustomize files? should we also update those in this PR so we at least have one good working Kube deploy?